backend/server.js

```diff
@@ -350,7 +350,17 @@ app.get('/search', wrap(async (req, res) => {
   const startTime = Date.now();
   const searchOutput = index.search(req.query.query, req.query.termId, minIndex, maxIndex);
   const midTime = Date.now();
-  const string = JSON.stringify(searchOutput.results);
+  let string;
+  if (searchOutput.analytics.wasSubjectMatch) {
+    string = JSON.stringify({
+      subjectCount: searchOutput.analytics.subjectCount,
+      subjectName: searchOutput.analytics.subjectName,
+      wasSubjectMatch: true,
+      results: searchOutput.results,
+    });
+  } else {
+    string = JSON.stringify({ results: searchOutput.results });
+  }
```

Comment:
if we changed the result object in backend/search.js could we change this entire block of new code to just 
```js
string = JSON.stringify({ results: searchOutput.resultsObject });
```